### DISCUSSION TITLE
How do I change biomes in the Bedrock Editor?

### DISCUSSION THREAD

**Discussion by TheMightyDarkness** [Mojang/minecraft-editor#210] [Q&A] [answered]
Is it possible to currently edit biomes by any chance in an area without external tools? Because /fillbiome currently does not exist on Minecraft Bedrock, how do I get to edit biome data within the Bedrock Editor? Is is possible to do this with a certain script?

**Comment by bryhaw** [comment, marked as answer]
We unfortunately don't support that in Editor yet! It is something we have discussed and I'm hoping we can prioritize it shortly since we have heard this feedback several times. :)

**Reply by TheMightyDarkness** [reply]
Thank you! For those wishing to edit biomes currently, you can use the Amulet Map Editor.
